proposed/phpdoc.md

```diff
@@ -1220,18 +1220,19 @@ The @method allows a class to know which 'magic' methods are callable.
 
 #### Syntax
 
-    @method [return type] [name]([type] [parameter], [...]) [description]
+    @method <static|> [name]([type] [parameter], [...])[: return type] [description]
```

Comment:
I'm hoping someone who knows how to express what I need will be able to add the syntax.